git-bin/Remotes/S3Remote.cs

```diff
@@ -206,11 +205,9 @@ public byte[] DownloadFile(string fileName, Action<int> progressListener)
             }
         }
 
-        private IAmazonS3 GetClient()
-        {
-            if (_client == null)
-            {
-                _client = AWSClientFactory.CreateAmazonS3Client(_key, _secretKey, _s3config);
+        private IAmazonS3 GetClient() {
+            if (_client == null) {
+                _client = new AmazonS3Client(_key, _secretKey, _s3config);

```

Comment:
The API changed slightly. Ref: https://docs.aws.amazon.com/AWSSdkDocsNET/V3/DeveloperGuide/net-dg-migration-guide-v3.html#awsclientfactory-removed